library/Mockery/Expectation.php

```diff
@@ -331,7 +340,7 @@ protected function _matchArg($expected, &$actual)
         if ($expected instanceof \Mockery\Matcher\MatcherAbstract) {
             return $expected->match($actual);
         }
-        if (is_a($expected, '\Hamcrest\Matcher') || is_a($expected, '\Hamcrest_Matcher')) {
+        if ($expected instanceof \Hamcrest\Matcher || $expected instanceof \Hamcrest_Matcher) {
```

Comment:
I'm sorry but I'm afraid that I don't see your point. I don't know what's the difference between using `is_a()` and using `instanceof` about this issue. If the target class wasn't present, `instanceof`would just return `false`, just like `is_a()` does, so I can't see the problem here. 

I agree with you about this isn't the topic of this PR, but I like to apply the boy scout rule when it is possible and I thought this was a good chance. Anyway, if there is any problem with that I can leave that commit out of this PR.